easybuild/easyconfigs/l/libsigc++/libsigc++-3.6.0-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
```

Comment:
We're not consistent with these attributions, some people add them, some don't, so imho that shouldn't block this PR.
These easyconfigs may have been created before the PR in which `libsigc++-3.4.0-GCCcore-11.3.0.eb` was created was opened, etc.

I know it's a difficult discussion, but unless we come up with a consistent way to do attributions, the Git log will have to serve as authoritative source I think...

That said: `libsigc++-3.6.0-GCCcore-11.3.0.eb` shouldn't be introduced, since there's `libsigc++-3.4.0-GCCcore-11.3.0.eb` already (which is used as a dependency for `airomm-1.16.2-GCC-11.3.0.eb`)